src/strategies_and_params.jl

```diff
@@ -176,12 +177,12 @@ Update the time step according to the strategy `s`.
 update_dτ(::ConstantTimeStep, dτ, args...) = dτ
 # here we implement the trivial strategy: don't change dτ
 
-"Slow down/Speed up `dτ` to control the psips overshoot."
 @with_kw mutable struct OvershootControl <: TimeStepStrategy
     targetwalkers::Int
     dτinit::Float64
     speedup::Bool
 end
+@doc "Slow down/Speed up `dτ` to control the psips overshoot." OvershootControl
```

Comment:
If you want to keep this code, can you please explain what it really does? Otherwise, do we still need it at alll?

Reply:
Yes it is no longer needed with the new critical damping shift. Commented out.